cursive-core/src/views/dialog.rs

```diff
@@ -167,6 +168,19 @@ impl Dialog {
         self.invalidate();
     }
 
+    /// Replace the content for this dialog.
+    ///
+    /// Previous content will be returned as a `BoxedView`.
+    pub fn replace_content<V: IntoBoxedView>(&mut self, view: V) -> BoxedView {
+        let old_content = replace(
```

Comment:
I think we can use the full name `std::mem::replace` here without importing it first.